neighbors/src/components/posts/single-post.jsx

```diff
@@ -0,0 +1,82 @@
+import { doc, getDoc } from 'firebase/firestore';
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { db } from '../../api/firebase';
+
+const PostWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  flex-wrap: wrap;
+`;
+const PostImg = styled.img`
+  width: 50%;
+  max-width: 500px;
+  object-fit: contain;
+  @media screen and (max-width: 767px) {
+    width: 100%;
+  }
+`;
+
+export const PostDetails = () => {
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
Nie rozumiem czym to steruje po co tutaj ten Loader jak on tutaj nie jest wykorzystywany do warunkowego renderowania? Do obgadania ta linijka. Dla mnie zbędne.

Reply:
Jest loader